src/components/Layout.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode} from 'react';
+import {SidebarProvider} from "@/components/ui/sidebar";
+import {AppSidebar} from "@/components/AppSidebar";
+import Header from "@/components/Header";
+
+type LayoutProps = {
+    children: ReactNode;
+};
+
+const Layout: React.FC<LayoutProps> = ({ children }) => {
+    return (
+        <SidebarProvider>
+            <AppSidebar />
+            <main className="w-screen h-screen">
+                <Header/>
```

Comment:
You are missing `<Dashboard />` component here in front of `<Header/>` (its there on the screenshot so I'd assume it should be included)

Reply:
I'm assuming that you are running locally, and it is missing at "/", but it should only show up during "/dashboard". Latest commit creates a temporary home page that hides sidebar and header.

![image](https://github.com/user-attachments/assets/0d266ada-1f60-4084-b225-d619141fabef)
![image](https://github.com/user-attachments/assets/69472ca7-8f91-493d-a0ff-3722decb6915)